File: twitter/src/main/java/com/twitterconnect/callback/TwException.java
package com.twitterconnect.callback;

import com.twitter.sdk.android.core.TwitterException;

/**
 * Created by amit on 4/2/17.
 */

public class TwException extends com.twitter.sdk.android.core.TwitterException {
    public TwException(String detailMessage) {
        super(detailMessage);
    }

    public TwException(String detailMessage, Throwable throwable) {
        super(detailMessage, throwable);
    }
}

File: tw1/src/main/kotlin/com/twitterconnect/callback/TwException.kt
package com.twitterconnect.callback

import com.twitter.sdk.android.core.TwitterException

/**
 * Created by amit on 4/2/17.
 */

class TwException : com.twitter.sdk.android.core.TwitterException {
    constructor(detailMessage: String) : super(detailMessage) {}

    constructor(detailMessage: String, throwable: Throwable) : super(detailMessage, throwable) {}
}
